src/main/java/org/kiwiproject/dropwizard/util/health/MongoHealthCheck.java

```diff
@@ -28,28 +28,29 @@ public MongoHealthCheck(DB db) {
     @SuppressWarnings({"deprecation", "ConstantConditions"})
     @Override
     protected Result check() throws Exception {
-        try {
-            var host = db.getMongo().getAddress().toString();
-            var dbName = db.getName();
+        var host = db.getMongo().getAddress().toString();
+        var dbName = db.getName();
 
-            var result = db.getStats();
+        try {
+            var statsCommandResult = db.getStats();
 
-            var isHealthy = result.ok();
+            var isHealthy = statsCommandResult.ok();
             var resultBuilder = newResultBuilder(isHealthy)
-                    .withDetail("storageSize", result.get("storageSize"))
-                    .withDetail("dataSize", result.get("dataSize"));
+                    .withDetail("storageSize", statsCommandResult.get("storageSize"))
+                    .withDetail("dataSize", statsCommandResult.get("dataSize"));
 
             if (isHealthy) {
                 return resultBuilder.withMessage("Mongo %s/%s is up", host, dbName).build();
             }
 
             return resultBuilder
                     .withMessage("Failed to retrieve db stats %s/%s : %s",
-                            host, dbName, result.getErrorMessage())
+                            host, dbName, statsCommandResult.getErrorMessage())
                     .build();
         } catch (Exception e) {
-            LOG.error("Unable to connect to Mongo: {}", e.getMessage(), e);
-            return newUnhealthyResult(HealthStatus.CRITICAL, "Mongo is not up: " + e.getMessage());
+            LOG.error("Unable to connect to Mongo: {}/{}", host, dbName, e);
+            return newUnhealthyResult(HealthStatus.CRITICAL, "Mongo %s/%s is not up: %s",
```

Comment:
Do we _know_ it is not up? Or maybe just not reachable? Suggest changing the unhealthy message to the same thing as the log message, since we cannot definitively say whether it is down, or up but not responding, or up but responding with an exception, etc.